maintenance_equipment_status/models/maintenance_equipment.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 ForgeFlow S.L. (https://forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class MaintenanceEquipment(models.Model):
+    _inherit = "maintenance.equipment"
+
+    status_id = fields.Many2one(
+        string="Status",
+        comodel_name="maintenance.equipment.status",
+        tracking=True,
+    )
+    status_name = fields.Char(
```

Comment:
Is this field necessary?

Reply:
`status_name` is used in equipment kanban view

Reply:
Yes, but im not sure if you could do `<t t-raw="record.statusid.name.raw_value" />` or something like this and delete the field

Reply:
OK, I understand... Locally tested, the module got installed, but a Javascript error is thrown when trying to load kanban view.
The changes made were
`<field name="status_id" />` instead of `<field name="status_name" /> `
and 
`record.status_id.name.raw_value` instead of `record.status_name.raw_value`

Reply:
Could yo try with `record.status_id.value`? I tested it on v13 and it worked

Reply:
it worked, thanks!
I'll safely remove that field, and then force push. @AdriaGForgeFlow is there any problem if `status_name` related field is removed?

Reply:
I think there should be no problem removing the field :+1: